ibtk/src/lagrangian/LSiloDataWriter.cpp

```diff
@@ -469,9 +468,9 @@ build_local_ucd_mesh(DBfile* dbfile,
 
     // Prune duplicate edges.
     std::set<std::pair<int, int> > local_edge_set;
-    for (auto it = edge_map.begin(); it != edge_map.end(); ++it)
+    for (const auto& map : edge_map)
```

Comment:
Is this a map?

Reply:
I'm not sure what an appropriate name would be. `edge_map` is of type `std::multimap<int, std::pair<int, int> >`. Perhaps `elem` for an element of the map? This kind of situation happens several times